test.py

```
@@ -0,0 +1,27 @@
+import os,sys 
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import

(F401)